app/build.gradle

```diff
@@ -31,9 +31,9 @@ dependencies {
     implementation project(':device')
     implementation project(':domain')
 
-
     testImplementation junit.junit
-
+    implementation "com.android.support:support-core-utils:27.1.1"
```

Comment:
1)Let's use androidx instead support library.
https://developer.android.com/topic/libraries/support-library/refactor

2) Use schema that I used for another dependencies.
All version and full names are in project build.gradle file.